### PR TITLE
Issue#75

### PR DESCRIPTION
**Pull request by DhritiShikhar**
* added ssh key support (ecdsa)
* updated validator.py
* modified help.py balloon and added warning

### CODE REVIEW

File: fas/validators.py [hunk 1]
```diff
@@ -233,8 +235,7 @@ def validate_python(self, value, state):
             if not keyline:
                 continue
             keyline = keyline.strip()
-            validline = re.match('^(rsa|ssh-rsa) [ \t]*[^ \t]+.*$', keyline)
-            if not validline:
+            if not self.valid_ssh_key:
```
Comment (ralphbean) [inline fas/validators.py]: Hm, this doesn't look right.  The regular expression isn't being applied anymore.

File: fas/validators.py [hunk 2]
```diff
@@ -233,7 +235,7 @@ def validate_python(self, value, state):
             if not keyline:
                 continue
             keyline = keyline.strip()
-            validline = re.match('^(rsa|ssh-rsa) [ \t]*[^ \t]+.*$', keyline)
+            validline = re.match(valid_ssh_key, keyline)
```
Comment (ralphbean) [inline fas/validators.py]: OK, so now the string that you have in the config:  ``"rsa,ssh-rsa,ecdsa,ecdsa-sha2"`` is a list of possible ssh key types, but it is not a regular expression.  I.e., it does not look like ``'^(rsa|ssh-rsa) [ \t]*[^ \t]+.*$'``.

**Comment by pypingou** [comment]
Maybe we should merge this manually to avoid all the merge commits at the top :-s

**Comment by laxathom** [comment]
@pypingou +1. So as PR #131 has all the commit already, I will close this one as duplicate.

File: fas/validators.py [hunk 3]
```diff
@@ -221,6 +221,9 @@ class ValidSSHKey(validators.FancyValidator):
     ''' Make sure the ssh key uploaded is valid '''
     messages = {'invalid_key': _('Error - Not a valid RSA SSH key: %(key)s')}
 
+    valid_ssh_key = config.get('valid_ssh_key').split(',').strip()
```
Comment (laxathom) [inline fas/validators.py]: why not just : 
>>> valid_ssh_key = config.get('valid_ssh_key').replace(',', '|').strip()

**Comment by laxathom** [comment]
Also, could you push the change to PR #131 as said above, this one will be closed as duplicate.

**Comment by ralphbean** [comment]
Closing as duplicate as suggested by @laxathom.

**Pull request closed without merging**